A2-service-configs-in-dns.md

```diff
@@ -0,0 +1,129 @@
+Service Config via DNS
+----------------------
+* Author(s): Mark D. Roth (roth@google.com)
+* Approver: a11r
+* Status: Draft
+* Implemented in: N/A
+* Last updated: 2017-01-19
+* Discussion at: https://groups.google.com/d/topic/grpc-io/DkweyrWEXxU/discussion
+
+## Abstract
+
+This document proposes a mechanism for encoding gRPC service config data
+in DNS for use in the open-source world.
+
+## Background
+
+The [service
+config](https://github.com/grpc/grpc/blob/master/doc/service_config.md)
+mechanism was originally designed for use inside of Google.  However,
+all but one part of the original design will work fine in the open-source
+world.  That one part is the specification of how the service config
+data will be encoded in DNS.
```

Comment:
I would suggest being a little more explicit in the second sentence. SOmething like .. "That one part is the specification of how the service config data will be encoded in DNS. This proposal fills-in this last missing piece."